niri-config/src/lib.rs

```diff
@@ -422,6 +422,8 @@ pub struct Gradient {
     pub angle: i16,
     #[knuffel(property, default)]
     pub relative_to: GradientRelativeTo,
+    #[knuffel(property, str)]
+    pub _in: GradientInterpolation,
```

Comment:
Prefixing with `_` means "allow unused"
```suggestion
    pub in_: GradientInterpolation,
```